incubator/java-openliberty/image/project/validate.sh

```diff
@@ -12,15 +12,6 @@ if [ ! -f ./pom.xml ]; then
   exit 1   
 fi
 
-# Get parent pom information (../pom.xml)
-args='export PARENT_GROUP_ID=${project.groupId}; export PARENT_ARTIFACT_ID=${project.artifactId}; export PARENT_VERSION=${project.version}
-export LIBERTY_GROUP_ID=${liberty.groupId}; export LIBERTY_ARTIFACT_ID=${liberty.artifactId}; export LIBERTY_VERSION=${version.openliberty-runtime}'
```

Comment:
How does removing this affect the tests later on in lines 69-73?

By using DOM naively we'd have a hard time taking into account profiles I'd think... are we sure we don't care?   I guess at a high level what were we actually checking before?

Reply:
We're no longer doing any of the checks from line 69 on (we skip those and `exit 0` earlier). We also pushed the liberty artifact configuration into the parent pom, so the child no longer has this anyways, making the check obsolete.

Reply:
I didn't realize that.  

What if we removed all that stuff and replaced with a check that the `version.liberty-maven-plugin` property hasn't been overridden?     This too would be more thorough a check if we checked from a maven context than just statically reading the pom.xml (since, assuming we add something like MVN_ARGS.. we could "cheat").  But it's maybe a good compromise with what this originally set out to do, and better than leaving this stuff in there?

Reply:
Good idea. I can add that in.